packages/core/admin/admin/src/pages/Settings/pages/Users/EditPage.tsx

```diff
@@ -177,13 +175,10 @@ const EditPage = () => {
     confirmPassword: '',
   } satisfies InitialData;
 
-  /**
-   * TODO: Convert this to react-query.
-   */
   const handleSubmit = async (body: InitialData, actions: FormikHelpers<InitialData>) => {
     lockApp?.();
 
-    const { confirmPassword, password, ...bodyRest } = body;
+    const { confirmPassword: _confirmPassword, password, ...bodyRest } = body;
```

Comment:
Do we need _confirmPassword ?

Reply:
No we want to exclude it from the API, eslint throws a warning for unused variables if they're not prefixed with _